test/hotspot/jtreg/gc/g1/TestLargePageUseForAuxMemory.java

```diff
@@ -126,6 +126,8 @@ static void testVM(String what, long heapsize, boolean cardsShouldUseLargePages,
         pb = ProcessTools.createJavaProcessBuilder(getOpts(heapsize, true));
 
         OutputAnalyzer output = new OutputAnalyzer(pb.start());
+        output.reportDiagnosticSummary();
```

Comment:
This should come last as if any `output.shouldxxx()` fails then the summary will be printed anyway, so we don't need to see it twice.